wallet-new/src/Cardano/Wallet/API/V1/Errors.hs

```diff
@@ -68,9 +69,13 @@ data WalletError =
     | WalletAlreadyExists
     | AddressNotFound
     | TxFailedToStabilize
+    | InvalidPublicKey { weProblem :: !Text }
+    | UnsignedTxCreationError
+    | SignedTxSubmitError { weProblem :: !Text }
     | TxRedemptionDepleted
     | TxSafeSignerNotFound { weAddress :: V1 Core.Address }
     | MissingRequiredParams { requiredParams :: NonEmpty (Text, Text) }
+    | CannotCreateAddress { weProblem :: !Text }
```

Comment:
I would much rather see more detail than a plain `Text`. `Text` doesn't give us much to go on. If these are mostly just wrapping underlying errors, then it's fine to keep them as text for now.